moto/dynamodb2/responses.py

```diff
@@ -126,24 +114,6 @@ def dynamodb_backend(self):
         """
         return dynamodb_backends[self.region]
 
-    @amz_crc32
-    @amzn_request_id
-    def call_action(self):
```

Comment:
Removed so that the BaseResponse super method is used instead. This enables default IAM support, but also changes the way self.body is handled, requiring the rest of the class to use the self._get_param helper method.

I'm not sure what the consequences of removing the @amz_crc32 and @amzn_request_id decorators is though.